### DISCUSSION TITLE
Ideas about misses sound in gameplay, settings menu

### DISCUSSION THREAD

**Discussion by vrgtics** [ppy/osu#28564] [Ideas]
# About misses sound in gameplay

As i know misses sound Not only when you miss the circle, but basically misses sound are weird, example when you miss 2 circles in a row, you dont hear it play the sound twice

- misses sound occurs when
   - you miss a circle
   - you suddenly out of the area of the slider after the head accuracy
   - you get bigger than 20 combos after you missed (hardcoded for some reason)

I think it can be implement just simply add a slider inside the settings section (ctrl+o) list, and the backend.

---
hopefully if the slider is set, considers where it should be on gameplay, rulesets, or audio main section, but i can see that one also call "audio" label inside the Gameplay section idky.. so let say **right over the upper area of "Beatmap" header** there, is a perfect to put it, i hope

the reason, a slider rather a toggle button, is that 20 combos default value "threshold" to make misses sound,
so i wish theres a left side and right side Between, set "20 combos (by default)"

   - so the slider property and label about smth like "Miss sound" in Gameplay audio section, 
     - if i slide that blob all the way to the Left, then the label should be "Always play the audio"
     - if i slide that blob all the way to the Right, then the label should be "Disabled"
     - if that blob in between cuz a slider we can set a range for let say "play after 1 to 100 combos" (i think if bigger than 100 combos is too large, is Not practical)

#
in terms for other Gamemodes, i believe its works as intended this can be able to apply for other Gamemodes too
also in current version, i noticed misses sound dont appear at the **very first circle in the beatmap level** when you intent to miss the circle, but i dont think will behave differently after the change

**Comment by n0Oo0Oo0b** [comment]
> I think it can be implement just simply add a slider inside the settings section (ctrl+o) list, and the backend.

Is there a specific reason you want this? Adding options just for the sake of it will just add clutter to the (already quite long) options menu.

While I agree that 20 combo is quite arbitrary, it works well enough and I don't see a reason why many people would want to change it.
Maybe if per-judgement hitsounds are added in the future (which I doubt) then it might be worth reconsidering. But as of right now, I don't see why this needs to be added.

Rulesets may be an argument since, for example, reaching 20 combo in Mania is much faster than in Standard. But I think this could be solved with better (ruleset-specific) defaults rather than a separate option.

>  also in current version, i noticed misses sound dont appear at the very first circle in the beatmap level when you intent to miss the circle, but i dont think will behave differently after the change

There's a "Always play first combo break" option (also in Gameplay > Audio) which changes that. If this setting does get added, then I agree this would be the most reasonable place to put it.
